doc/changes/latest.inc

```diff
@@ -79,6 +79,8 @@ Enhancements
 
 - Added support for Aurora version 2021.9.0.6 to :func:`mne.io.read_raw_nirx` (:gh:`10668` by `Robert Luke`_)
 
+- Add timeformat ``YYYY/MM/DD`` to ``mne.io.ctf.info._convert_time``  (:gh:`10674` by `Simon Kern`)
```

Comment:
Missing an underscore after your name (see the entry above for an example)

Reply:
No, I already contributed (minoryl, just like this), but my name is missing from the authors.inc

thanks! I'll add that